examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb

```diff
@@ -1,670 +1,764 @@
+        "\n",
+        "\n",
+        "We define the statistical measure of predictive bias function, $score_{bias}(S)$ as a likelihood ratio score and a function of a given subgroup $S$. The null hypothesis is that the given prediction's odds are correct for all subgroups in $\\mathcal{D}$:\n",
+        "\n",
+        "$$H_{0}:odds(y_{i})=\\frac{\\hat{p}_{i}}{1-\\hat{p}_{i}}\\ \\forall i\\in\\mathcal{D}.$$\n",
+        "\n",
+        "The alternative hypothesis assumes some constant multiplicative bias in the odds for some given subgroup $S$:\n",
+        "\n",
+        "$$H_{1}:\\ odds(y_{i})=q\\frac{\\hat{p}_{i}}{1-\\hat{p}_{i}},\\ \\text{where}\\ q>1\\ \\forall i\\in S\\ \\mathrm{and}\\ q=1\\ \\forall i\\notin S.$$\n",
+        "\n",
+        "In the classification setting, each observation's likelihood is Bernoulli distributed and assumed independent. This results in the following scoring function for a subgroup $S$:\n",
+        "\n",
+        "\\begin{align*}\n",
+        "score_{bias}(S)= & \\max_{q}\\log\\prod_{i\\in S}\\frac{Bernoulli(\\frac{q\\hat{p}_{i}}{1-\\hat{p}_{i}+q\\hat{p}_{i}})}{Bernoulli(\\hat{p}_{i})}\\\\\n",
+        "= & \\max_{q}\\log(q)\\sum_{i\\in S}y_{i}-\\sum_{i\\in S}\\log(1-\\hat{p}_{i}+q\\hat{p}_{i}).\n",
+        "\\end{align*}\n",
+        "Our bias scan is thus represented as: $S^{*}=FSS(\\mathcal{D},\\mathcal{E},F_{score})=MDSS(\\mathcal{D},\\hat{p},score_{bias})$.\n",
+        "\n",
+        "where $S^{*}$ is the detected most anomalous subgroup, $FSS$ is one of several subset scan algorithms for different problem settings, $\\mathcal{D}$ is a dataset with outcomes $Y$ and discretized features $\\mathcal{X}$, $\\mathcal{E}$ are a set of expectations or 'normal' values for $Y$, and $F_{score}$ is an expectation-based scoring statistic that measures the amount of anomalousness between subgroup observations and their expectations.\n",
+        "\n",
+        "Predictive bias emphasizes comparable predictions for a subgroup and its observations and Bias scan provides a more general method that can detect and characterize such bias, or poor classifier fit, in the larger space of all possible subgroups, without a priori specification."
       ]
-     },
-     "execution_count": 3,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "(X_test, X_train,\n",
-    " y_test, y_train) = train_test_split(X, y, test_size=3694, shuffle=True, random_state=0)\n",
-    "\n",
-    "X_train.head()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 4,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "array(['Recidivated', 'Survived'], dtype=object)"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Install AIF360\n",
+        "!pip install 'aif360[LawSchoolGPA]'"
```

Comment:
same here: we don't need LawSchoolGPA

Reply:
I got the message below, so added LAwSchoolGPA
WARNING:root:No module named 'tempeh': fetch_lawschool_gpa will be unavailable. To install, run:
pip install 'aif360[LawSchoolGPA]'

The warning was resolved after adding aif360[LawSchoolGPA]

Reply:
It's just a warning but it's not relevant to this code so we can ignore it.

Reply:
Changes updated